.github/workflows/build.yaml

```diff
@@ -52,3 +52,50 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  wheel-build-pylibraft:
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux-build.yml@main
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
+      repo: rapidsai/raft
```

Comment:
linking to this comment to keep things DRY

- https://github.com/rapidsai/cuml/pull/5109#discussion_r1063938257